src/client/client.js

```diff
@@ -0,0 +1,26 @@
+const client = {
+  handleGetWishlists: () =>
+    fetch('http://localhost:3001/wishlist', {
+      method: 'GET',
```

Comment:
The fetch API defaults requests to GET unless otherwise specified, so this line can be omitted.